### PR TITLE
VACMS 4110: Improve content release integration with advancedqueue.

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #4110. 

The `va_gov_build_trigger` module is really ungainly, internally inconsistent, and filled with technical debt, so I'm going to try and beat that down while dealing with the primary issue.  Expect tests to fail in droves until I'm done.

## Testing done
Adding behavioral tests, maybe others as needed.

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue #4110:
    - [ ] users can see the status of all builds on the Content Release page (Tugboat/local) regardless of how it was initiated
    - ~every environment uses the queue for FE build~ (will be done as part of #4617).
    - [ ] advancedqueue runner checks for in-process web builds and requeues the task if any are encountered or if the build has errored
    - [ ] builds complete successfully regardless of how they were initiated
    - [ ] check if the state is removed when job errors
    - [ ] split composer commands into (1) runner (2) queuer

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behavioral/cypress/integration/content_release.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Content Release
+  In order to reliably and predictably release content
+  As anyone involved in the project
+  I need the content release page to manage and reflect an orderly underlying release process
+
+  @content_release @content_release_page
+  Scenario: The content release page should normally display no in-process releases
+    Given I am logged in as a user with the "content_admin" role
+    And I am at "/admin/content/deploy"
+    Then I should see "No recent updates"
```
Comment (acrollet) [inline tests/behavioral/cypress/integration/content_release.feature:10]: 🤔 Does this stand a chance of failing if someone logs into their new preview and kicks off a content release right away?

Comment (ndouglas) [reply tests/behavioral/cypress/integration/content_release.feature:10]: I'm not married to this test sticking around; it's mostly just there as a basis for further tests.

That said... we run tests on preview environments?  I knew we did on dev/staging BRD, but I hadn't considered that we'd run them on previews.  If people are going to be kicking off content releases, and we're trying to have an orderly queue, then it seems kinda savage for me to just stomp in and start scattering their content releases to the winds and substituting my own 😂

Comment (indytechcook) [reply tests/behavioral/cypress/integration/content_release.feature:10]: Demo environments do not run tests but PR environments do.

**Comment by ndouglas** [comment]
Is there a reason we're running these Drush commands in this way?

Screen Shot 2021-03-04 at 9 15 11 AM

As far as I can tell, that TRUE/FALSE isn't actually doing anything:

Screen Shot 2021-03-03 at 12 43 41 PM

A third positional argument [isn't mentioned in Drush 8](https://drushcommands.com/drush-8x/state/state-set/) and supplying one is treated as an error in Drush 10:

Screen Shot 2021-03-04 at 9 39 55 AM

I'm guessing that the `va_gov.build ` was accidentally inserted.

**Comment by ndouglas** [comment]
Part of the refactoring work is pulling out the command generation a bit so that we can get a clearer look at what will be executed.
Screen Shot 2021-03-05 at 12 26 00 PM

**Comment by ndouglas** [comment]
Closing in favor of #4620.

**Pull request closed without merging**